code/controllers/subsystem/job.dm

```diff
@@ -30,6 +30,21 @@ SUBSYSTEM_DEF(job)
 	var/list/crew_obj_list = list()
 	var/list/crew_obj_jobs = list()
 
+	/// jobs that are not allowed in HoP job manager
+	var/list/manipulation_blacklist_jobs = list(
```

Comment:
This needs a better name, something like `locked_jobs` or `read_only_jobs`

Reply:
Those suggestions can misinform that the jobs are not available in the game due to "locked_flags" name existence, but these jobs are still usable.
I will pick `job_manager_blacklisted`